activerecord/test/active_record/connection_adapters/fake_adapter.rb

```diff
@@ -32,7 +32,9 @@ def merge_column(table_name, name, sql_type = nil, options = {})
           name.to_s,
           options[:default],
           fetch_type_metadata(sql_type),
-          options[:null])
+          options[:null],
+          table_name,
```

Comment:
This test wan't passing the table name, resulting in a quite unrealistic scenario (IMHO)